.github/workflows/fix-eof-newline-coderabbit-docstrings.yml

```
@@ -0,0 +1,106 @@
+          git status --porcelain
+          git add -A
+
+          git config user.name "github-actions[bot]"
+          git config user.email "github-actions[bot]@users.noreply.github.com"
+
+          git commit -m "Fix missing final newline in docstring PR"
+
+      - name: Push fix commit back to PR branch
+        if: steps.guard.outputs.run == 'true'
+        shell: bash
+        run: |
```

actionlint (1.7.10)

"github.event.pull_request.head.ref" is potentially untrusted. avoid using it directly in inline scripts. instead, pass it through an environment variable. see https://docs.github.com/en/actions/reference/security/secure-use#good-practices-for-mitigating-script-injection-attacks for more details

(expression)